operations/kubernetes/Containerd.md

```diff
@@ -1,26 +1,26 @@
-# Containerd
+# `Containerd`
 
-Containerd is a daemonset that runs on the host. It is used to run containers on the Kubernetes platform.
+`Containerd` is a `daemonset` that runs on the host. It is used to run containers on the Kubernetes platform.
 
-### /var/lib/containerd filling up
+## `/var/lib/containerd` filling up
 
-In older versions of containerd, there are cases where the `/var/lib/containerd` directory fills up. In the event that this occurs, the following steps can be used to remediate the issue.
+In older versions of `containerd`, there are cases where the `/var/lib/containerd` directory fills up. In the event that this occurs, the following steps can be used to remediate the issue.
 
-1. Restart containerd on the NCN:
+1. Restart `containerd` on the NCN:
 
    ```bash
    ncn-w001 # systemctl restart containerd
    ```
 
-   Many times this will free up space in `/var/lib/containerd` -- if not proceed to Step 2.  See notes below for subsequent steps that must be taken after containerd is restarted (independent of disk space issues).
+   Many times this will free up space in `/var/lib/containerd` -- if not proceed to Step 2.  See notes below for subsequent steps that must be taken after `containerd` is restarted (independent of disk space issues).
```

Comment:
```suggestion
   Many times this will free up space in `/var/lib/containerd` -- if not, then proceed to the next step.
   In either case, see the notes in the [Restart `containerd`](#restart-containerd) section for subsequent steps that must be taken after `containerd` is restarted (independent of disk space issues).
```